_includes/catalog-index-band.html

```diff
@@ -0,0 +1,229 @@
+<div class="grid-wrapper component extensions-page">
```

Comment:
extensions catalog won't be ready for the launch - lets remove/comment it out for now.

Reply:
I submitted a PR to remove the extensions section but I don't see it.